src/convertToObject.js

```diff
@@ -5,8 +5,21 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const arrStyles = sourceString.split(';').map((str) => str.trim());
+  const arrByArrStyles = arrStyles.map((str) => {
+    const parts = str.split(':');
+
+    return parts.map((part) => part.trim());
+  });
+  const newObj = {};
+
+  arrByArrStyles.forEach((element) => {
+    newObj[element[0]] = element[1];
```

Comment:
This line assumes that each element has exactly two parts (key and value). Consider adding validation to ensure this assumption holds, or handle cases where it doesn't.